chevah/github_hooks_server/handler.py

```diff
@@ -317,6 +317,9 @@ def issue_comment(self, event):
             action='issue_comment',
             )
 
+        if commenter_name.endswith('[bot]'):
```

Comment:
Hm... is this how github advertises the bot accounts?

I was expecting to have a separate attribute to signal that the user is a bot

![image](https://github.com/chevah/github-hooks-server/assets/204609/de380202-291f-4f30-b410-78a3a44376be)

![image](https://github.com/chevah/github-hooks-server/assets/204609/0010c6b5-4523-43e7-a954-c5ba9b2ec179)

Reply:
Yes, there is also a User "type" field which I should probably use, but also the login has a [bot] suffix in the API, and it was easier this way.